Lib/glyphsLib/builder/features.py

```diff
@@ -89,6 +89,11 @@ def _to_ufo_features(font, ufo=None, generate_GDEF=False, skip_export_glyphs=Non
         if feature.notes:
             lines.append("# notes:")
             lines.extend("# " + line for line in feature.notes.splitlines())
+            feature_name = re.search("^Name: (.+)", feature.notes, flags=re.MULTILINE)
+            if feature_name and len(feature_name.groups()) == 1:
```

Comment:
it's unlikely that you get more than one match, but in that case I'd simply take the last one

Reply:
Thanks! Changed in a5fb108.